batavia/types/Float.js

```diff
@@ -379,6 +382,10 @@ Float.prototype.__sub__ = function(other) {
         } else {
```

Comment:
At column 71: [(no-trailing-spaces) Trailing spaces not allowed.](http://eslint.org/docs/rules/no-trailing-spaces)